server/test/unit/plugins/loader/test_api.py

```diff
@@ -252,3 +252,56 @@ def test_catalogers(self):
     def test_finalize(self):
         api.finalize()
         self.assertEqual(api._MANAGER, None)
+
+
+class TestAPIModels(unittest.TestCase):
+
+    @mock.patch('pulp.plugins.loader.api._is_initialized')
+    @mock.patch('pulp.plugins.loader.api._MANAGER')
+    def test_get_unit_model(self, mock_manager, mock_is_initialized):
```

Comment:
Consider renaming to match literal function name being tested:
`def test_get_unit_model` = `def test_get_unit_model_by_id`